specification/search/data-plane/Azure.Search/stable/2023-11-01/searchindex.json

```diff
@@ -0,0 +1,2059 @@
+          "x-ms-client-name": "SearchText"
+        },
+        "searchFields": {
+          "type": "string",
+          "description": "The comma-separated list of field names to which to scope the full-text search. When using fielded search (fieldName:searchExpression) in a full Lucene query, the field names of each fielded search expression take precedence over any field names listed in this parameter."
+        },
+        "searchMode": {
+          "$ref": "#/definitions/SearchMode",
+          "description": "A value that specifies whether any or all of the search terms must be matched in order to count the document as a match."
+        },
+        "select": {
+          "type": "string",
+          "description": "The comma-separated list of fields to retrieve. If unspecified, all fields marked as retrievable in the schema are included."
+        },
+        "skip": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The number of search results to skip. This value cannot be greater than 100,000. If you need to scan documents in sequence, but cannot use skip due to this limitation, consider using orderby on a totally-ordered key and filter with a range query instead."
+        },
+        "top": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The number of search results to retrieve. This can be used in conjunction with $skip to implement client-side paging of search results. If results are truncated due to server-side paging, the response will include a continuation token that can be used to issue another Search request for the next page of results."
+        },
+        "semanticConfiguration": {
+          "type": "string",
+          "description": "The name of a semantic configuration that will be used when processing documents for queries of type semantic."
+        },
+        "semanticErrorHandling": {
+          "$ref": "#/definitions/SemanticErrorHandling",
+          "description": "Allows the user to choose whether a semantic call should fail completely (default / current behavior), or to return partial results."
+        },
+        "semanticMaxWaitInMilliseconds": {
+          "type": "integer",
+          "format": "int32",
+          "x-nullable": true,
+          "minimum": 700,
+          "description": "Allows the user to set an upper bound on the amount of time it takes for semantic enrichment to finish processing before the request fails."
+        },
+        "answers": {
+          "$ref": "#/definitions/Answers",
+          "description": "A value that specifies whether answers should be returned as part of the search response."
+        },
+        "captions": {
+          "$ref": "#/definitions/Captions",
+          "description": "A value that specifies whether captions should be returned as part of the search response."
+        },
+        "semanticFields": {
```

Comment:
@clareluna could you check if you need to include this? I thought it might be missing so I put it here.

Reply:
Thanks @robertklee -- we are actually not GAing the semantic fields version of semantic ranker. I will revert this line tomorrow AM